man/systemd-fstab-generator.xml

```diff
@@ -71,6 +71,16 @@
     for more information about special <filename>/etc/fstab</filename>
     mount options this generator understands.</para>
 
+    <para>One special topic is handling of symbolic links. Since systemd v216,
+    mount units refuse mounting when the target <varname>Where=</varname> is a
+    symbolic link. Until systemd v234, any paths in
+    <filename>/etc/fstab</filename> that named symbolic links were translated
+    literally into mount units, and hence would fail. But to enhance backwards
+    compatibility, <filename>systemd-fstab-generator</filename> now resolves
+    symlinks to the extent it can. If a symlink target does not exist at the
+    time that this generator runs, it is assumed that the symlink target is the
+    final target of the mount.</para>
```

Comment:
Humm, we generally avoid documenting legacy behaviour in current man pages. People are expected to use current man with current versions. Hence I think we really should only document the new behaviour in the man pages.

I see the usefulness of having docs about old behaviour, but we haven't done that so far, and it tends get easily out-of-date, hence I think we shouldn't do this here either. I hope that makes sense.